<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>104</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>0.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0.05</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0.01</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>bonferroni</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sidak</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>holm-sidak</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>holm</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>simes-hochberg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>hommel</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>fdr_bh</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>fdr_by</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>fdr_tsbh</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>fdr_tsbky</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>104</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
